docs/cli_reference/dfdaemon.md

```diff
@@ -13,22 +13,23 @@ dfdaemon [flags]
 ### Options
 
 ```
-      --callsystem string   caller name (default "com_ops_dragonfly")
-      --certpem string      cert.pem file path
-      --dfpath string       dfget path (default "/go/src/github.com/dragonflyoss/Dragonfly/bin/linux_amd64/dfget")
-  -h, --help                help for dfdaemon
-      --hostIp string       dfdaemon host ip, default: 127.0.0.1 (default "127.0.0.1")
-      --keypem string       key.pem file path
-      --localrepo string    temp output dir of dfdaemon (default "/root/.small-dragonfly/dfdaemon/data")
-      --maxprocs int        the maximum number of CPUs that the dfdaemon can use (default 4)
-      --notbs               not try back source to download if throw exception (default true)
-      --port uint           dfdaemon will listen the port (default 65001)
-      --ratelimit string    net speed limit,format:xxxM/K
-      --registry string     registry addr(https://abc.xx.x or http://abc.xx.x) and must exist if dfdaemon is used to mirror mode (default "https://index.docker.io")
-      --rule string         download the url by P2P if url matches the specified pattern,format:reg1,reg2,reg3
-      --urlfilter string    filter specified url fields (default "Signature&Expires&OSSAccessKeyId")
-      --verbose             verbose
-  -v, --version             version
+      --callsystem string     caller name (default "com_ops_dragonfly")
+      --certpem string        cert.pem file path
+      --dfpath string         dfget path (default "/go/src/github.com/dragonflyoss/Dragonfly/bin/linux_amd64/dfget")
```

Comment:
I do not think this default value is correct. Am I right? @lowzj @Starnop @zhouhaibing089

Reply:
The default value is the path where `dfdaemon` located in.

Reply:
If users have installed dfdaemon, do we have a default path for daemon?
If there is, then I think it is OK to add that path in the document.
Otherwise, it may bring some misleading in the document for users.

Reply:
The installing command `make install` we provided to install `dfget` and `dfdameon` as a whole `dfclient` in directory `/op/dragonfly` and create soft links in `/usr/local/bin`.

I think we can use `dfdaemon gen-doc` command after `make install`, then the default value of `--dfpath` in the document would be `/usr/local/bin/dfget`.

Reply:
> use `dfdaemon gen-doc` command after `make install`

Actually I have done this:
```
"CliDocGeneratorCmd": "make build && make install && make docs"
```

If we dig further, we will found that it is due to https://github.com/dragonflyoss/Dragonfly/blob/master/hack/generate-docs.sh#L14-L15. The code will use the built binary to generate docs rather than the installed binaries.

@fengzixu Could we first use installed binaries, and then use built binaries if the previous ones do not exist?

Reply:
Okay. I will reviese it. @allencloud